src/style.css

```
@@ -1 +1,65 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body,
+ul {
+  margin: 0;
+}
+
+html {
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+  align-items: center;
+  padding: 0 50px;
+  --color: #00acdc;
```

give more meaningful naming like
```suggestion
  --active-color: #00acdc;
```